static/app/views/issueList/addViewPage.tsx

```diff
@@ -92,6 +93,67 @@ function AddViewPage({savedSearches}: {savedSearches: SavedSearch[]}) {
   );
 }
 
+function AddViewBanner({hasSavedSearches}: {hasSavedSearches: boolean}) {
+  const organization = useOrganization();
+
+  const {isPromptDismissed, dismissPrompt} = usePrompt({
+    feature: 'issue_views_add_view_banner',
```

Comment:
Have you added this to the backend? There should be a list of constants that the backend will validate from

Reply:
Ah I did not, making a PR for this now